app/controllers/meetings_controller.rb

```diff
@@ -45,72 +26,52 @@ def comment
 
   # rubocop:disable MethodLength
   def delete_comment
-    comment_exists = Comment.where(id: params[:commentid]).exists?
-    is_my_comment = Comment.where(
-      id: params[:commentid],
-      comment_by: current_user.id
-    ).exists?
+    comment = Comment.find_by(id: params[:commentid])
+    is_my_comment = comment.present? && (comment.comment_by == current_user.id)
 
-    if comment_exists
-      meeting_id = Comment.where(id: params[:commentid]).first.commentable_id
-      is_my_meeting = MeetingMember.where(
-        meeting_id: meeting_id,
-        user_id: current_user.id,
-        leader: true
-      ).exists?
-      is_member = MeetingMember.where(
-        meeting_id: meeting_id,
-        user_id: current_user.id
-      ).exists?
-    else
-      is_my_meeting = false
-      is_member = false
+    if comment.present?
+      meeting_id = comment.commentable_id
+      meeting = Meeting.find_by(id: meeting_id)
+      is_my_meeting = MeetingMember.find_by(
+        id: current_user.id, leader: true
+      )
+      is_member = meeting.members.find_by(id: current_user).present?
     end
 
-    if comment_exists && ((is_my_comment && is_member) || is_my_meeting)
-      CommentNotificationsService.remove(comment_id: params[:commentid],
-                                         model_name: 'meeting')
+    if comment.present? && ((is_my_comment && is_member) || is_my_meeting)
```

Comment:
same here, can we extract this boolean statement into it's own method so this is a bit more clear as to the intent of this